docs/testing/ie-11-testing.md

```diff
@@ -1,44 +1,50 @@
 ---
-ms.date: 05/16/2020
-description: 'Test your Office Add-in using Internet Explorer 11.'
 title: Internet Explorer 11 testing
+description: 'Test your Office Add-in on Internet Explorer 11.'
+ms.date: 05/19/2021
 localization_priority: Normal
 ---
 
-# Test your Office Add-in using Internet Explorer 11
+# Test your Office Add-in on Internet Explorer 11
 
-Depending on the specifications of your add-in, you may plan to support older versions of Windows and Office, which require testing on Internet Explorer 11. This is often necessary as part of submitting your add-in to AppSource. You can use the following command line tooling to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing.
+If you plan to market your add-in in AppSource or you plan to support certain older versions of Windows and Office, you must test the add-in using the embeddable web view control that is based on Internet Explorer 11 (called "Trident"). You can use a command line to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing. For information about which versions of Windows and Office use the Internet Explorer 11 web view control, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
 
-## Pre-requisites
+> [!IMPORTANT]
+> Internet Explorer 11 does not support JavaScript versions later than ES5. If you want to use the syntax and features of ECMAScript 2015 or later, you have two options:
+>
+> - Write your code in ECMAScript 2015 (also called ES6) or later JavaScript, or in TypeScript, and then compile your code to ES5 JavaScript using a compiler such as [babel](https://babeljs.io/) or [tsc](https://www.typescriptlang.org/index.html).
+> - Write in ECMAScript 2015 or later JavaScript, but also load a [polyfill](https://en.wikipedia.org/wiki/Polyfill_(programming)) library such as [core-js](https://github.com/zloirock/core-js) that enables IE to run your code.
+>
+> For more information about these options, see [Support Internet Explorer 11](../develop/support-ie-11.md).
+>
+> Also, Internet Explorer 11 does not support some HTML5 features such as media, recording, and location.
+
+> [!NOTE]
+> To test how your add-in runs in Office on the web in Internet Explorer, simply open Office on the web in Internet Explorer and [sideload the add-in](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md).
```

Comment:
Some suggestions for brevity and clarity about the browser testing (instead of the embedded browser control).
```suggestion
> To test your add-in on the Internet Explorer 11 browser, open Office on the web in Internet Explorer and [sideload the add-in](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md).
```